spec/classes/nodejs_spec.rb

```diff
@@ -122,13 +122,13 @@
           end
         end
 
-        context 'and repo_pin not set' do
```

Comment:
The original statement is probably more accurate.  If you set something to `undef`, you're not actually setting it.  The parameter default gets used (which happens to be `undef`).